src/main/analytics/webdav-handlers.ts

```diff
@@ -48,39 +73,59 @@ function subscribeToFlowEvents(ipc: IpcWebdavFlow) {
   ipc.on('WEBDAV_FILE_RENAMED', (_, payload) => {
     const { name } = payload;
 
+    broadcastToWindows('sync-info-update', {
+      action: 'RENAMED',
+      name,
```

Comment:
we are not missing the type here?

Reply:
on this one, the name comes with extension, so i've renamed it for a better understanding